Source/Scene/Scene.js

```diff
@@ -1650,35 +1650,67 @@ define([
         }
     }
 
-    function executeShadowMapCommands(scene) {
-        var context = scene.context;
-        var uniformState = context.uniformState;
-        var shadowMap = scene.sunShadowMap;
-        var renderState = shadowMap.renderState;
-        var passState = shadowMap.passState;
-
-        // Clear shadow depth
-        shadowMap.clearCommand.execute(context);
-
-        // Render from the shadow's camera
-        uniformState.updateCamera(shadowMap.camera);
-
-        var shadowMapCommands = scene._frustumCommandsList[0]; // TODO : temporary solution
-        if (defined(shadowMapCommands)) {
-            // TODO : Execute only opaque and translucent commands for now.
+    function getShadowMapCommands(scene) {
+        // TODO : temporary solution for testing
+        var shadowMapCommands = [];
+        var frustumCommands = scene._frustumCommandsList[0];
+        if (defined(frustumCommands)) {
             var startPass = Pass.GLOBE;
             var endPass = Pass.TRANSLUCENT;
             for (var pass = startPass; pass <= endPass; ++pass) {
-                var commands = shadowMapCommands.commands[pass];
-                var length = shadowMapCommands.indices[pass];
+                var commands = frustumCommands.commands[pass];
+                var length = frustumCommands.indices[pass];
                 for (var i = 0; i < length; ++i) {
                     var command = commands[i];
                     if (defined(command.shadowCastProgram)) {
-                        executeCommand(command, scene, context, passState, renderState, command.shadowCastProgram);
+                        shadowMapCommands.push(command);
                     }
                 }
             }
         }
+        return shadowMapCommands;
+    }
+
+    function executeShadowMapCommandsNoCascades(scene, context, uniformState, shadowMap, renderState, passState) {
```

Comment:
It was for comparison, but I can remove it and use the other code instead (looping over 1 cascade).